File: app/src/main/java/kr/ac/korea/translator/view/common/BaseActivity.java
package kr.ac.korea.translator.view.common;

import android.app.Activity;
import android.content.Context;
import android.os.Bundle;
import android.view.View;

public class BaseActivity extends Activity implements View.OnClickListener {
    protected String token;
    @Override
    public void onClick(View v) {
    }

    @Override
    protected void onCreate( Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    public void init(){

    }

    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(newBase);
    }


    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: app/src/main/java/kr/ac/korea/translator/view/common/BaseActivity.kt
package kr.ac.korea.translator.view.common

import android.app.Activity
import android.content.Context
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

open class BaseActivity : Activity(), View.OnClickListener {
    protected var token: String? = null
    override fun onClick(v: View?) {}
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    fun init() {}
    override fun attachBaseContext(newBase: Context?) {
        super.attachBaseContext(newBase)
    }

    override fun onDestroy() {
        super.onDestroy()
    }
}